src/IO/Observer/ReductionActions.hpp

```diff
@@ -92,19 +92,34 @@ struct ContributeReductionDataToWriter;
  * reduction data and used to print an informative message when the reduction is
  * complete. The formatter must conform to
  * `observers::protocols::ReductionDataFormatter`.
+ *
+ * This action also supports observing the intermediate stage of the reduction
```

Comment:
Maybe make it clear that the data is written both per core and per simulation? If I've understood the code correctly at least :)

Reply:
added a note to the docs